build.gradle

```diff
@@ -23,7 +23,7 @@ ext {
     bfdVersion='1.3.1'
     aggregatorVersion='1.2.1'
     filtersVersion='1.6.1'
-    eventClientVersion='1.3.1'
+    eventClientVersion='1.3.2'
```

Comment:
This is a code change, so it's: 1.4.0. If next week we have to do dependabots, then it will be 1.4.1.

Reply:
changed the version